### PR TITLE
[4.0]namespace administrator components com-templates

### PR DESCRIPTION
**Pull request by brianteeman**
continuing the work to namespace joomla4 one admin component at a time.

please wait for drone to successfully complete before reviewing.

### CODE REVIEW

File: administrator/components/com_templates/View/Template/HtmlView.php [hunk 1]
```diff
@@ -125,7 +130,7 @@ class HtmlView extends BaseHtmlView
 	 */
 	public function display($tpl = null)
 	{
-		$app            = \JFactory::getApplication();
+		$app            = Factory::getApplication();
 		$this->file     = $app->input->get('file');
 		$this->fileName = \JFilterInput::getInstance()->clean(base64_decode($this->file), 'string');
```
Comment (C-Lodder) [inline administrator/components/com_templates/View/Template/HtmlView.php]: `JFilterInput` missed

Comment (brianteeman) [reply administrator/components/com_templates/View/Template/HtmlView.php]: thanks - fixed

File: administrator/components/com_templates/Model/TemplateModel.php [hunk 2]
```diff
@@ -15,6 +15,12 @@
 use Joomla\CMS\MVC\Model\FormModel;
 use Joomla\CMS\Plugin\PluginHelper;
 use Joomla\Component\Templates\Administrator\Helper\TemplateHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Fileystem\File;
```
Comment (Quy) [inline administrator/components/com_templates/Model/TemplateModel.php]: Spelling `Fileystem`

File: administrator/components/com_templates/Model/StyleModel.php [hunk 3]
```diff
@@ -399,17 +402,17 @@ protected function preprocessForm(\JForm $form, $data, $group = 'content')
 	{
```
Comment (Quy) [inline administrator/components/com_templates/Model/StyleModel.php:403]: Change `\JForm` to `Form`

File: administrator/components/com_templates/tmpl/style/edit_options.php [hunk 4]
```diff
@@ -9,20 +9,23 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\HTML\HTMLHelper;
+
 // Load chosen.css
```
Comment (Quy) [inline administrator/components/com_templates/tmpl/style/edit_options.php:15]: Remove comment?

Comment (brianteeman) [reply administrator/components/com_templates/tmpl/style/edit_options.php:15]: not as part of this pr

Comment (wojsmol) [reply administrator/components/com_templates/tmpl/style/edit_options.php]: remove `+` from lines 12-14

**Comment by wilsonge** [comment]
Thanks guys!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**